hawc/apps/common/helper.py

```diff
@@ -289,3 +289,51 @@ def build_export(self) -> FlatExport:
 
 
 re_digits = r"\d+"
+
+
+def find_matching_list_element_value_by_value(
```

Comment:
The kwargs are being used in GroupResult::flat_complete_row_data. (I did remove get_p_value_qualifier_display, at some point I think I had it doing something different but yea that is just the built-in behavior so that could go).

Reply:
Did the serializer change here? I'd just write a separate function to handle this, since it's really doing a different thing

Reply:
for me it was failing on the test_epi_api.py::test_full_export method. By the time flat_complete_data_row is called it's just getting a a dictionary of values so for instance a p_test_qualifier of "-" was showing up as "-" and not "n.s". (but maybe that's actually desirable and the test needs to change to match?)

Reply:
It's because the serializer changed. I f we change the serializer to `ser["p_value_qualifier_display"]` does that do it?

Reply:
yup, that works. OK, renamed/simplified the function.